docs/vscode_setup.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Using VSCode
+sidebar_position: 5
```

Comment:
This markdown is correct however this sidebar_position (5) would now be a dupliate. Can you update the `ado_pipelines_example.md`, `nested_CTES.md`, and `contributing.md` sidebar_positions to avoid this duplication? Otherwise I believe ties are settled alphabetically.